govcd/vapp_network.go

```diff
@@ -80,6 +80,10 @@ func (vapp *VApp) GetVappNetworkById(id string, refresh bool) (*types.VAppNetwor
 
 	util.Logger.Printf("[TRACE] Looking for networks: %s --- %d", id, len(vapp.VApp.NetworkConfigSection.NetworkConfig))
 	for _, vappNetwork := range vapp.VApp.NetworkConfigSection.NetworkConfig {
+		// break early for empty network interfaces. They don't have all information
+		if vappNetwork.NetworkName == "none" {
```

Comment:
Not a must, but could be 
```suggestion
		if vappNetwork.NetworkName == types.NoneNetwork {
```

Reply:
changed